README.md

```diff
@@ -5,6 +5,20 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/openshift/assisted-service)](https://goreportcard.com/report/github.com/openshift/assisted-service)
 [![License Apache](https://img.shields.io/github/license/openshift/assisted-service)](https://opensource.org/licenses/Apache-2.0)
 
+## About
```

Comment:
Just a proposal: maybe add references to some of the recent publicly available demo videos here

Reply:
Good idea, done.